examples/service-clients/azure-client/todo-list/src/fluid.ts

```diff
@@ -0,0 +1,164 @@
+import {
+	type ContainerSchema,
+	type IFluidContainer,
+	SharedTree,
+	TreeViewConfiguration,
+} from "fluid-framework";
+import { SharedString } from "fluid-framework/legacy";
+import { v4 as uuid } from "uuid";
+
+import { AzureFunctionTokenProvider } from "./azureFunctionTokenProvider.js";
+import { TodoItem, TodoList } from "./schema.js";
+
+// Define the server we will be using and initialize Fluid
+const useAzure = process.env.FLUID_CLIENT === "azure";
+
+const user = {
+	id: uuid(),
+	name: uuid(),
+};
+
+const azureUser = {
+	id: user.id,
+	name: user.name,
+	additionalDetails: {
+		gender: "female",
+		email: "xyz@microsoft.com",
+	},
+};
+
+/**
+ * Azure Fluid Relay connection configuration.
+ */
+export const connectionConfig: AzureRemoteConnectionConfig | AzureLocalConnectionConfig =
+	useAzure
+		? {
+				type: "remote",
+				tenantId: "",
+				tokenProvider: new AzureFunctionTokenProvider("", azureUser),
+				endpoint: "",
+			}
+		: {
+				type: "local",
+				tokenProvider: new InsecureTokenProvider("fooBar", user),
+				endpoint: "http://localhost:7070",
+			};
+
+/**
+ * Schema for the TODO list application.
+ * @remarks
+ * This includes the DataObjects we support and any initial DataObjects we want created
+ * when the Container is first created.
+ */
+export const todoListContainerSchema = {
+	initialObjects: {
+		todoList: SharedTree,
+	},
+	dynamicObjectTypes: [SharedString],
+} as const satisfies ContainerSchema;
+/**
+ * Container schema type for the Todo List application.
+ */
+export type TodoListContainerSchema = typeof todoListContainerSchema;
+
+const treeViewConfig = new TreeViewConfiguration({
+	schema: TodoList,
+	enableSchemaValidation: true,
+});
+
+/**
+ * Loads the application model from an existing Fluid Container.
+ */
+export function loadAppFromExistingContainer(
+	container: IFluidContainer<TodoListContainerSchema>,
+): TodoList {
+	const tree = container.initialObjects.todoList;
+	const treeView = tree.viewWith(treeViewConfig);
+	if (!treeView.compatibility.canView) {
+		throw new Error("Expected container data to be compatible with app schema");
+	}
+	return treeView.root;
+}
+
+/**
+ * Initializes the application model for a newly created Fluid Container.
+ * @remarks
+ * Includes population of the initial TodoList contents.
+ */
+export async function initializeAppForNewContainer(
+	container: IFluidContainer<TodoListContainerSchema>,
+): Promise<TodoList> {
+	const tree = container.initialObjects.todoList;
+	const treeView = tree.viewWith(treeViewConfig);
+	if (!treeView.compatibility.canInitialize) {
+		throw new Error("Expected container data to be compatible with TodoList schema");
+	}
+
+	const initialTodoList = await createInitialTodoList(container);
+	treeView.initialize(initialTodoList);
+
+	return treeView.root;
+}
+
+async function createInitialTodoList(
+	container: IFluidContainer<TodoListContainerSchema>,
+): Promise<TodoList> {
+	const listTitle = await container.create(SharedString);
```

Comment:
Similar here and below in createTodoItem(), consider tutorial comments highlighting the learning objective of this demo